client/src/templates/Introduction/SuperBlockIntro.js

```diff
@@ -148,11 +148,14 @@ class SuperBlockIntroductionPage extends Component {
     const nodesForSuperBlock = edges.map(({ node }) => node);
     const blockDashedNames = uniq(nodesForSuperBlock.map(({ block }) => block));
     const i18nSuperBlock = t(`intro:${superBlock}.title`);
+    const i18nCertText = t(`intro:misc-text.certification`, {
+      cert: i18nSuperBlock
```

Comment:
```suggestion
      cert: superBlock
```

Reply:
Hey @ShaunSHamilton, thank you for the suggestion.

If I remember correctly, `superBlock` here is the dashed version of the super block, and is only in English (responsive-web-design).

To get the titles to render correctly in all languages, we need to get the localized version of the title first, then pass it as the variable `cert` to `misc-text.certification` in the `intro.json` file.

That should give us `Responsive Web Design Certification | freeCodeCamp.org`, `Certificación de Diseño Web Responsivo | freeCodeCamp.org`, and so on.

Reply:
Right. My bad, I forgot we used an object to pass variables 🤦 , and not to search for properties.

Reply:
No worries! i18next is a whole other layer of complexity on top over everything else we've got going on. Thanks again for taking a look at this.